From 884130cc0d01bfbae94e5f88fce475840f3f6b52 Mon Sep 17 00:00:00 2001
From: Pritesh Patel <pritesh.patel@einfochips.com>
Date: Thu, 29 Aug 2024 15:42:10 +0530
Subject: [PATCH 040/128] riscv: dts: hifive-premier-p550: Remove memory nodes

Memory size will be passed through kernel command line based on
the RAM detected by U-Boot

Signed-off-by: Pritesh Patel <pritesh.patel@einfochips.com>
---
 .../boot/dts/eswin/eic7700-hifive-premier-p550.dts   | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
index e6e7ef8cb688..c6229169924f 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
@@ -57,18 +57,6 @@ cpus {
 		timebase-frequency = <RTCCLK_FREQ>;
 	};
 
-	memory@59000000 {
-		device_type = "memory";
-		reg = <0x0 0x59000000 0x0 0x400000>;
-		numa-node-id = <0>;
-	};
-
-	memory@80000000 {
-		device_type = "memory";
-		reg = <0x0 0x80000000 MEMORY_SIZE_H MEMORY_SIZE_L>;
-		numa-node-id = <0>;
-	};
-
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
2.47.0

